dist/tools/kconfiglib/genconfig.py

```diff
@@ -53,12 +64,34 @@ def is_module(symbol):
     """
     return symbol.name.startswith("MODULE_")
 
+
 def is_error(symbol):
     """
     Checks if a given symbol represents an error, depending on its prefix.
     """
     return symbol.name.startswith("ERROR_")
 
+
+def log(message, level=logging.DEBUG, color=None):
+    """
+    Logs a message using 'logging', with a given color and level. If no level is
+    passed the message is logged using debug level. If no color is passed the
+    following rules apply:
+        - error messages are RED
+        - warning messages are YELLOW
+        - all other messages have no color
+    """
+    if color is None:
+        if level == logging.ERROR:
+            color = Colors.RED
+        elif level == logging.WARNING:
+            color = Colors.YELLOW
+        else:
+            color = Colors.RESET
+
+    logging.log(level, "{}{}{}".format(color, message, Colors.RESET))
```

Comment:
These should probably be disabled if not running on a terminal, similar to `color.inc.mk`